File: WebAdmin/Models/Local.cs
using System;
using System.Collections.Generic;


namespace WebAdmin.Models
{
    public class Local
    {
        public int Id {get; set;}
        public int Type {get; set;}
        public string Name {get; set;}
        public int Floor {get; set;}
        public DateTime OpeningTime {get; set;}
        public DateTime ClosingTime {get; set;}
    }
}

File: WebAdmin/src/Models/Local.js
class Local {
    Id;
    Name;
    Type;
    Floor;
    Admins;

    constructor(name = '', type = -1, floor = -1, id = -1, admins = []) {
      this.Id = id;
      this.Name = name;
      this.Type = type;
      this.Floor = floor;
      this.Admins = admins;
    }
  }